src/v/archival/tests/service_test.cc

```diff
@@ -9,21 +9,16 @@
  */
 
 #include "archival/ntp_archiver_service.h"
-#include "archival/service.h"
 #include "archival/tests/service_fixture.h"
 #include "cluster/commands.h"
 #include "net/unresolved_address.h"
 #include "storage/types.h"
 #include "test_utils/async.h"
 
 #include <seastar/core/deleter.hh>
-#include <seastar/core/file.hh>
 #include <seastar/core/future-util.hh>
 #include <seastar/core/lowres_clock.hh>
-#include <seastar/core/metrics.hh>
-#include <seastar/core/smp.hh>
 #include <seastar/core/sstring.hh>
-#include <seastar/http/request.hh>
 #include <seastar/testing/thread_test_case.hh>
 #include <seastar/util/log.hh>
```

Comment:
these are unrelated to the PR, but these are all unused imports and too small to open a new PR over.